templates/user_profile_page.hbs

```diff
@@ -241,119 +241,162 @@
     {{/is}}
 
     {{#isnt current_filter.identifier 'activities'}}
-      <div class="container">
-        <section class="profile-section">
-
-          {{#if contributions}}
-
-            <header class="profile-section-header">
-              <h2 class="profile-section-title">{{t current_filter.identifier}}</h2>
-              {{#if sorters}}
-                <span class="profile-section-description">{{sorter_description}}</span>
-                <span class="profile-section-sorter dropdown">
-                  <button class="dropdown-toggle" aria-haspopup="true">
-                    {{current_sorter.label}}
-                    <svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" focusable="false" viewBox="0 0 12 12" class="dropdown-chevron-icon">
-                      <path fill="none" stroke="currentColor" stroke-linecap="round" d="M3 4.5l2.6 2.6c.2.2.5.2.7 0L9 4.5"/>
-                    </svg>
-                  </button>
-                  <span class="dropdown-menu" role="menu">
-                    {{#each sorters}}
-                      <a href="{{url}}" aria-selected="{{selected}}" role="menuitem">
+      {{#is current_filter.identifier 'badges'}}
+        <div class="container">
+          <section class="profile-section">
+
+            {{#if user.badges}}
+              <header class="profile-section-header">
+                <h2 class="profile-section-title">{{t 'badges'}}</h2>
+                <span class="profile-section-description">{{t 'badges_description' name=user.name}}</span>
+              </header>
+
+              <div class="profile-badges-items">
+                {{#each user.badges}}
+                  <div class="profile-badges-item">
+                    <div class="profile-badges-item-image">
+                      <img class="badge" src="{{icon_url}}">
+                    </div>
+                    <div>
+                      <div class="profile-badges-item-title">
                         {{name}}
-                      </a>
-                    {{/each}}
-                  </span>
-                </span>
-              {{/if}}
-            </header>
-
-            <ul class="profile-contribution-list profile-{{current_filter.identifier}}">
-              {{#each contributions}}
+                      </div>
+                      <p class="profile-badges-item-description">
+                        {{description}}
+                      </p>
+                    </div>
+                    <div class="profile-badges-item-metadata">
+                      <div class="profile-badges-item-metadata-title">
+                        {{t 'badges_awarded'}}
+                      </div>
+                      <p class="profile-badges-item-metadata-description">
+                        {{date assigned_at format='medium'}}
```

Comment:
The date is a bit confusing for screen reading; can you add an `aria-label` with something like “This badge was awarded on {{date}}”?

Reply:
What is the best way to do it? Ideally, I'd want to have a translation for the string. However, how am I going to pass in the formatted date? This might be a good use case for subexpressions like 👇 😄 
```
{{ t 'badges_awarded_label' date=(date assigned_at format='medium')}}
```

cc @luis-almeida

Reply:
We agreed to wait with this until https://github.com/zendesk/curlybars/pull/22 is merged.